webapp/channels/src/types/store/views.ts

```diff
@@ -23,6 +24,10 @@ export type AdminConsoleUserManagementTableProperties = {
     sortColumn: string;
     sortIsDescending: boolean;
     pageSize: number;
+    pageIndex: number;
+    userId: string;
+    columnValue: string;
```

Comment:
can we call this something with cursor to differentiate this is for cursor purpose